Makefile

```diff
@@ -5,11 +5,17 @@ ccflags-y := -std=gnu99 -Wno-declaration-after-statement
 KDIR ?= /lib/modules/$(shell uname -r)/build
 GIT_HOOKS := .git/hooks/applied
 
-all:
+all: kmod blocklist-util
+
+kmod:
 	$(MAKE) -C $(KDIR) M=$(shell pwd) modules
 
+blocklist-util: blocklist_user.c
```

Comment:
Rename to `vwifi-tool`. At present, only block list management is supported, and we can introduce more features later.

Reply:
Ok, I'll rename it. Thank you.